api_tests/src/actors/actor.ts

```diff
@@ -782,20 +782,14 @@ export class Actor {
             | "REFUNDED"
             | "INCORRECTLY_FUNDED"
     ) {
+        // This is hackery of the higest order.
```

Comment:
Shouldn't exactly this PR allow to remove this hackery? We are returning the state now.

Reply:
This function is rcf003 specific but is called by code that uses the new protocols i.e., `bob.fund()` etc.  The test code needs detangling.